torchvision/models/__init__.py

```diff
@@ -8,8 +8,8 @@
 from .mobilenet import *
 from .mnasnet import *
 from .shufflenetv2 import *
-from .efficientnet import *
```

Comment:
Can you add this like back?

Reply:
Yep on it :) Sorry!

Reply:
Nope, there's more to do. Need to add a leaf module that can't be traced through from effnet. Will send it soon